packages/@n8n/nodes-langchain/nodes/llms/LMChatAnthropic/LmChatAnthropic.node.ts

```diff
@@ -22,6 +22,26 @@ const modelField: INodeProperties = {
 	type: 'options',
 	// eslint-disable-next-line n8n-nodes-base/node-param-options-type-unsorted-items
 	options: [
+		{
+			name: 'Claude Sonnet 4.5',
+			value: 'claude-sonnet-4-5-20250929',
+		},
+		{
+			name: 'Claude Sonnet 4',
+			value: 'claude-sonnet-4-20250514',
+		},
+		{
+			name: 'Claude Opus 4.1',
+			value: 'claude-opus-4-1-20250805',
+		},
+		{
+			name: 'Claude Opus 4',
+			value: 'claude-opus-4-20250514',
+		},
+		{
+			name: 'Claude Haiku 4.5',
+			value: 'claude-haiku-4-5-20251001',
+		},
 		{
```

Comment:
These options are for legacy version of the node, before node was fetching models dynamically. So we don't need to maintain list of models manually